bindings/node/test/autoEncrypter.test.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+const fs = require('fs');
+const BSON = new (require('bson'))();
```

Comment:
request: Add `//TODO: upgrade to 4.x bson`
q: should we also be testing against the 4.x bson? or is that overkill?

Reply:
I think until we move to bson 4.x in the driver, it's okay not to test here.